setup/generate_usptream_rust_keylime_code_coverage/test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+# for Packit PRs we would be uploading code coverage files unless forbidden
+[ -n "${PACKIT_SOURCE_URL}" -a -z "${UPLOAD_COVERAGE}" ] && UPLOAD_COVERAGE=1
+UPLOAD_URL=https://transfer.sh
+
+
+rlJournalStart
+
+    rlPhaseStartSetup "Collect code coverage for rust components"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        #delete coverage script, in dir are mandatory only coverage files
+        rlRun "pushd ${__INTERNAL_limeCoverageDir}"
+        # export in format for user
+        rlRun "grcov . --binary-path /usr/local/bin/keylime_agent -s . -t html --branch --ignore-not-existing -o e2e_coverage.html"
+        # export coverage report in format compatible for codecov
+        rlRun "grcov . --binary-path /usr/local/bin/keylime_agent -s . -t lcov --branch --ignore-not-existing -o e2e_testsuite.txt"
```

Comment:
I am confused. If these are the same data just with a different format why don't you call it e2e_coverage.txt?